.github/workflows/examples/rank-math-pro.yml

```diff
@@ -0,0 +1,89 @@
+name: Updater
+
+# Run this workflow twice daily
+on: 
+  # schedule:
+  #   - cron: '0 */12 * * *'
+  workflow_dispatch:
+
+env:
+  PACKAGE_SLUG: seo-by-rank-math-pro
+
+jobs:
+  # Set the job key. The key is displayed as the job name
+  # when a job name is not provided
+  update:
+    # Name the Job
+    name: Fetch and check versions
+    # Set the type of machine to run on
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # Fetch latest version
+      - name: Fetch
+        run: wget 'https://rankmath.com/download/510590/?username=${{secrets.RANK_MATH_USERNAME}}&api_key=${{secrets.RANK_MATH_PRO_KEY}}' -O package.zip
+
+      # PHP
+      - name: 'Setup PHP'
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+        
+      # Install dependencies
+      - name: Composer
+        run: 'composer install'
+
+      # Get current tag
+      - name: 'Get Previous tag'
+        id: previous_version
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        
+      # Get new version
+      - name: 'Get version'
+        id: new_version
+        run: echo "new_version=$(php .github/workflows/check.php)" >> $GITHUB_OUTPUT

```

Comment:
I believe this is the last step that requires _this_ project's dependencies, so the `vendor` folder can probably be removed here in a new step so that its contents don't get merged with those from Rank Math Pro's `vendor` directory. To ensure the final product best reflects what Rank Math Pro sends over.